packages/nuget-exe/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "nuget-exe",
+  "version": "4.9.1",
+  "description": "Repackaged NuGet.exe bianry",
+  "main": "nuget-4.9.2.exe",
```

Comment:
It would be nifty if we could move NuGet logic into this package to offer a JS NuGet wrapper of high quality, but packaging the raw EXE still seems like a positive step for now.

Reply:
No need to change, just sharing idea for the future... but perhaps it will be more extensible in the future if we had a default export of the path from a is file, that way we could deal with windows vs Linux/Mac (even though for some reason they don't ship an executable yet and have it baked into msbuild tasks for xplat.....

Reply:
Yeah, I definitely think having a JS wrapper that exposes NuGet functionality would be cleaner. Seems like something we should consider if we spend some more time in the area.